node/pkg/alephium/reobserve.go

```diff
@@ -144,7 +144,7 @@ func (w *Watcher) getGovernanceEventsByTxId(
 
 		reobservedEvents = append(reobservedEvents, &reobservedEvent{
 			&event,
-			maxUint8(msg.consistencyLevel, w.minConfirmations),
+			msg.consistencyLevel,
```

Comment:
Not related to this PR: how do we enforce the minimum consistencyLevel for new messages from users?

Reply:
Currently, we enforce the minimum consistency level in both frontend and contract:

1. The frontend sets the `consistencyLevel` to 105 when the user sends a transaction
2. We also check in the contract that the `consistencyLevel` for the transfer transaction is not less than 105